.github/workflows/python-publish.yml

```diff
@@ -14,34 +6,16 @@ on:
 
 jobs:
   deploy:
-
     runs-on: ubuntu-latest
-
     steps:
-    - uses: actions/checkout@v4
-    - name: Set up Python
-      uses: actions/setup-python@v5
-      with:
-        python-version: '3.x'
-    - name: Assert tag version matches __version__ attribute
```

Comment:
I think we should keep this assertion. Unless this is now magically handled otherwise but I don't see it